pulumi/src/api.ts

```diff
@@ -48,15 +50,14 @@ export async function deployApi(args: DeployApiArgs): Promise<k8s.apps.v1.Deploy
   if (config.api === undefined) return
 
   const tier = 'api'
-  const labels = { app, asset, tier }
-  const name = `${asset}-${tier}`
-  const [coinstack] = asset.split('-')
+  const labels = { coinstack, assetName, tier }
```

Comment:
we lost `app` label. also `app` -> `appName` and `asset` -> `assetName` create a diff across all api resources and no longer match labels in `statefulService`. I don't find the `Name` suffix super useful, but I will let you make the call there and make sure all labels reflect the same naming convention with this update

Reply:
On the k8s level I agree it's pointless, but on TS level I do find it useful to differentiate objects from strings. I've aligned it so that we only modify the TS vars

Reply:
I typically just rely on the associated type, but I'm ok with leaving for more explicit clarity